### ISSUE TITLE
Element IDs are not unique (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#140]
This may be a more complex developer issue. IDs generated in the HTML are repeated and IDs should be unique. 
2,986 occurrences
--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [ ] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow. 

PS This one might be more complicated. If it's not clear how to fix this, let me know ASAP and we will work on Plan B.

**Comment by tubesoft** [comment]
I think this comes from TEI coding. `@xml:id` is copied to `@id` in html. I think change the TEI code should work. For example adding the document number to the folio number should work. (From `id="thirtythreer-ref-1"` to `id="ALCH00034-thirtythreer-ref-1"`)

TEI:
<img width="669" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/fd37684f-411f-4eaf-bfbc-660b589367a4">

Rendered:
<img width="860" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/1bb5b09f-f85d-4a11-9ef0-87ed54261291">

**Comment by mdalmau** [comment]
@tubesoft : Is this something that needs to be batch updated in the TEI files?

**Comment by tubesoft** [comment]
@mdalmau I think so. Just adding the ALCH number to the beginning of the id would work.

**Comment by mdalmau** [comment]
@tubesoft : I am going to need you to talk me through this issue. I am not sure I fully understand. I'll follow-up over email to schedule a 30 min meeting with you and probably @aewingate too. Alex is pretty familiar with the encoding conventions from updating the encoding guide + all the bibl encoding. I just want to make sure that changing the ID values do not impact something else.

**Comment by aewingate** [comment]
@tubesoft and @mdalmau, I think that as long as the xml:id and anywhere it is referenced with the "#" change, we're okay. But happy to sit in on whatever meeting. And then I can make changes in the encoding guide as needed

**Comment by mdalmau** [comment]
@aewingate : I totally forgot that @tubesoft ID'd this as a TEI issue. Can you revisit when you have a chance? We can schedule 30 mins to go over it later this week. Thanks!

**Comment by aewingate** [comment]
@mdalmau: I know we'll confirm in a meeting, but is this just on `<ref>` or also the `<milestone>` for example? If there are 2,986 instances, I'm inclined to think this is an issue with `<milestone>`, though the editorial annotations through `<ref>` could also get pulled in if someone (like me) uses the same naming convention for their editorial annotations across manuscripts.

**Comment by mdalmau** [comment]
We do not need to fix this according to SiteImprove. If it pops up again in a scan, Michelle will submit an exception explaining how IDS are unique within each manuscript encoding. We use an XML editor to validate !
